tests/fuzz/s2n_certificate_extensions_parse_test.c

```diff
@@ -51,9 +51,6 @@ static const uint8_t TLS_VERSIONS[] = {S2N_TLS13};
 
 int s2n_fuzz_init(int *argc, char **argv[])
 {
-    /* Initialize the trust store */
```

Comment:
If s2n_fuzz_init is empty, I would just delete it for this test. You can input NULL into the S2N_FUZZ_TARGET() macro.